apps/example/src/domain/invoice-auto-emailer.ts

```diff
@@ -5,47 +5,56 @@ import z from "zod"
 import { EmailSender, ISendEmails } from "./send-email.js"
 import * as Config from "../config/equinox.js"
 import * as Invoice from "./invoice.js"
+import { Context } from "../context/context.js"
 
-const CATEGORY = "InvoiceAutoEmail"
-const streamId = StreamId.gen(InvoiceId.toString)
+export namespace Stream {
```

Comment:
wait what happened to CategoryId?

Reply:
I was waiting on you to land it in dotnet templates 😊

Reply:
I did and I CC'd you a few days back (though the docs are lagging, as they sometimes do)

Reply:
Oh, my bad then, I’m on holiday right now and was travelling last week. I’ll get on it 💪